frontend/src/components/dashboard/page_timings/PageTimingsTable.vue

```diff
@@ -0,0 +1,174 @@
+<template>
+    <VContainer
+        p-0
+        class="users-table"
+    >
+        <VRow
+            class="header my-3"
+            fluid
+        >
+            <VCol>
+                <VSelect
```

Comment:
maybe you can use default dropdown that we have in common folder? (if not, it's ok)

Reply:
i thinks it's a different dropdown, the default one is for time period this one is for the parameter to sort values by

Reply:
Drop-down that I was referring is placed in components/common it receives items as option

Reply:
i'm gonna stick with this one, the default dropdown contains styling that we use for the period dropdown, this one's different